expyfun/stimuli/_tracker.py

```diff
@@ -779,46 +805,25 @@ def _pick(self):
             inds = active
         return inds[self._rand.randint(len(inds))]
 
-    def get_trial(self):
-        """Selects the tracker from which the next trial should be run
-
-        Returns
-        -------
-        subscripts : list-like
-            The position of the selected tracker.
-        x_current : float
-            The level of the selected tracker.
-        """
-        if not self._trial_complete:
-            # Chose a new tracker before responding, so record non-response
-            self._response_history = np.append(self._response_history,
-                                               np.nan)
-        self._trial_complete = False
-        self._current_tracker = self._pick()
-        self._tracker_history = np.append(self._tracker_history,
-                                          self._current_tracker)
-        ss = np.unravel_index(self._current_tracker, self.shape)
-        level = self._trackers.flat[self._current_tracker].x_current
-        self._x_history = np.append(self._x_history, level)
-        return ss, level
-
     def respond(self, correct):
         """Update the current tracker based on the last response
 
         Parameters
         ----------
         correct : boolean
             Was the most recent subject response correct?
-
-        Notes
-        -----
-        ``get_trial`` must be run before ``respond`` can be called.
         """
         if self._trial_complete:
             raise RuntimeError('You must get a trial before you can respond.')
         self._trackers.flat[self._current_tracker].respond(correct)
         self._trial_complete = True
         self._response_history = np.append(self._response_history, correct)
+        if self.stopped:
+            self._callback(
+                'dealer_%i_stop' % self._dealer_id, json.dumps(dict(
+                    tracker_position=[int(s) for s in self._tracker_history],
```

Comment:
@Eric89GXL @drammock since what is stored in _tracker_history is actually subscripts, this will obviously bomb for any matrix of trackers with more than one dimension. Is there a better/functional way to format the subscripts?